lib/platform/github/index.js

```diff
@@ -1018,6 +1018,7 @@ async function mergePr(prNo, branchName) {
   config.baseCommitSHA = null;
   // Delete branch
   await deleteBranch(branchName);
+  await delay(5000);
```

Comment:
The logic for this needs to be moved from platform to inside lib/workers:

1. mergePr() is not the only way we automerge. We also automerge branches too sometimes
2. We may use mergePr() for *other* types of merges one day, and maybe we don't want the delay

please locate the places in branch and pr where the automerges are done and add the delay there instead.